srcpkgs/dracut/files/kernel-uefi-hook-postrm

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+#
+# Kernel post-remove hook for dracut EFI bundle.
+#
+# Arguments passed to this script: $1 pkgname, $2 version.
+#
+PKGNAME="$1"
+VERSION="$2"
+
+. "${ROOTDIR}/etc/default/dracut-uefi-hook"
+if [ "x${CREATE_UEFI_BUNDLES}" != x1 ]; then
```

Comment:
I don't think we need this check.
Just check if the file exist then remove it?

Reply:
Agree.